### PR TITLE
Update config_decryptor_aes_with_iv.py

### PR DESCRIPTION
**Pull request by doomedraven**

### CODE REVIEW

File: src/rat_king_parser/config_parser/utils/decryptors/config_decryptor_aes_with_iv.py [hunk 1]
```diff
@@ -60,14 +60,16 @@ class ConfigDecryptorAESWithIV(ConfigDecryptor):
...
+        # b"\x18": ECB,
+        # b"\x19": OFB,
+        b"\x1A": CFB8,
     }
 
     # Patterns for identifying AES metadata
-    _PATTERN_AES_KEY_AND_BLOCK_SIZE_AND_ALGO = re.compile(
-        b"[\x06-\x09]\x20(.{4})\x6f.{4}[\x06-\x09]\x20(.{4})\x6f.{4}[\x06-\x09](.)\x6f.{4}",
-        re.DOTALL,
+    _PATTERN_AES_KEY_AND_BLOCK_SIZE_AND_ALGO = (
```
[the remark this answers is not in the document]
Comment (doomedraven) [reply src/rat_king_parser/config_parser/utils/decryptors/config_decryptor_aes_with_iv.py:70]: OR works and PRed

File: src/rat_king_parser/config_parser/utils/decryptors/config_decryptor_aes_with_iv.py [hunk 2]
```diff
@@ -217,17 +219,20 @@ def _get_aes_key_block_size_and_algo(
         self,
     ) -> Tuple[int, int, ModeWithInitializationVector]:
         logger.debug("Extracting AES key and block size...")
-        hit = re.search(
-            self._PATTERN_AES_KEY_AND_BLOCK_SIZE_AND_ALGO, self._payload.data
-        )
-        if hit is None:
+        hit = False
```
[the remark this answers is not in the document]
Comment (doomedraven) [reply src/rat_king_parser/config_parser/utils/decryptors/config_decryptor_aes_with_iv.py:222]: sorry i confused PR's answering previous quesiton about OR, no or doesn't work here, just tested

File: src/rat_king_parser/config_parser/utils/decryptors/config_decryptor_aes_with_iv.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (doomedraven) [reply src/rat_king_parser/config_parser/utils/decryptors/config_decryptor_aes_with_iv.py:70]: ok it works, i had some typo on first test

**Comment by jeFF0Falltrades** [comment]
👍

**Pull request merged**